### ISSUE TITLE
If DnaSampleFormatEnum and RnaSampleFormatEnum have the same PVs, could they be be reformulated as a singe enum?

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/submission-schema#77]
enum | permissible_value
-- | --
DnaSampleFormatEnum | 10 mM Tris-HCl
DnaSampleFormatEnum | DNAStable
DnaSampleFormatEnum | Ethanol
DnaSampleFormatEnum | Low EDTA TE
DnaSampleFormatEnum | MDA reaction buffer
DnaSampleFormatEnum | PBS
DnaSampleFormatEnum | Pellet
DnaSampleFormatEnum | RNAStable
DnaSampleFormatEnum | TE
DnaSampleFormatEnum | Water

enum | permissible_value
-- | --
RnaSampleFormatEnum | 10 mM Tris-HCl
RnaSampleFormatEnum | DNAStable
RnaSampleFormatEnum | Ethanol
RnaSampleFormatEnum | Low EDTA TE
RnaSampleFormatEnum | MDA reaction buffer
RnaSampleFormatEnum | PBS
RnaSampleFormatEnum | Pellet
RnaSampleFormatEnum | RNAStable
RnaSampleFormatEnum | TE
RnaSampleFormatEnum | Water

cc @mslarae13

**Comment by turbomam** [comment]
see also
- #48

**Comment by mslarae13** [comment]
@turbomam  yes, they can be consolidated. We had them listed separately because of the common slots in DH, but that's to be resolved. so the same enum can be used now.

**Comment by mslarae13** [comment]
@pkalita-lbl can you confirm.. We had to keep the slots separate. RNA vs DNA. But the permissible values can share the same enum. Yes?

**Comment by pkalita-lbl** [comment]
Yes, we could have one enum and both the DNA and RNA slots' ranges could point to that enum. No problem there.

**Comment by ssarrafan** [comment]
Removing this one from the sprint (so I can close the sprint) and adding the backlog label.  Let me know if should be in the new/current sprint @mslarae13

**Comment by mslarae13** [comment]
Going to consider this de-prioritized for the berk-schema refactor roll out. Will work on this soon as berk is in production.
FYI @pkalita-lbl @ssarrafan @turbomam

**Comment by ssarrafan** [comment]
@mslarae13 is there still a plan to work on this?

**Comment by mslarae13** [comment]
yes this still should be done. It's schema clean up.

**Comment by mslarae13** [comment]
Resolved as part of https://github.com/microbiomedata/nmdc-schema/pull/2607
